### ISSUE TITLE
GUI Scaling options (Font size)

### ISSUE THREAD

**Issue by kilian102201** [files-community/Files#3954]
**Is your feature request related to a problem? Please describe.**
This is not related to a specific problem so far.

**Describe the solution you'd like**
I would love to be able to resize my fontsize for the overall app gui. 
Right now we have the option for a light and dark theme but this extra options gives me even more possible tweaks to fine tune my personal preferences.

![image](https://user-images.githubusercontent.com/80431760/110715135-f6fa2200-8204-11eb-9e85-fac9c350afeb.png)

**Describe alternatives you've considered**
The is no alternative for this request.

**Additional context**
Add any other context or screenshots about the feature request here. Ex: What device type/UI layout and app version?
I'm using the latest version of Files 1.3.0.0. 
I'm running on Windows 10 Pro version 20H2 Build 19042.844 (Windows Feature Experience Pack)
Processor	Intel(R) Core(TM) i9-8950HK CPU @ 2.90GHz  -  RAM-Memory 32,0 GB (31,9 GB Available)

**Comment by yaichenbaum** [comment]
@kilian102201 The next version of Files will allow you to override theme resources including the font sizes in Files. You can find instructions here https://files-community.github.io/docs/#/articles/custom-themes.

**Comment by kilian102201** [comment]
@yair100 I've read the article: https://files-community.github.io/docs/#/articles/custom-themes
This is about a custom theme regarding color schemes.
I'm mainly interested in changes the scaling of the text in an overal principle and that's not mentioned in this color scheming template article? So i'm guessing this is something that has been left out. Besides that I would love to not dive too deep into coding because i'm mainly interested in being an end user and not so much into diving into the code itself. It may be very simple to spot how to acces everything for an average person that reads/writes allot of code but for an average end user/consumer it's not something that's very accesible. 

Do you maybe happen to have any comments regarding the things I've mentioned above?

**Comment by kilian102201** [comment]
Overriding theme resources sounds great but would be even nicer to have an option available in the options menu of the program itself to change the font size or the scaling of the complete application itself. Nothing fancy but very easy to use for anyone without diving into the code itself.

**Comment by yaichenbaum** [comment]
@winston-de Can you help us with instructions on how to override font sizes from the resource Files?

**Comment by kilian102201** [comment]
> @winston-de Can you help us with instructions on how to override font sizes from the resource Files?

I'm not great with diving deep into code so if it could be as simply as possible that would be appreciated :)

**Comment by winston-de** [comment]
> @winston-de Can you help us with instructions on how to override font sizes from the resource Files?

@yair100 you should be able to create a new style for text blocks and text boxes (without a resource key so that it applies to all by default) and set the font size there. However, if I recall correctly, many of the font sizes are hard coded so this may not work effectively.

```xaml
<Style TargetType="TextBlock">
	<Setter Property="FontSize" Value="10" />
</Style>
```

**Comment by winston-de** [comment]
@kilian102201 some other solutions to your request is to change the default font size in windows or change the scaling. You can find these under Settings > ease of access > make text bigger and settings > ease of access > make everything bigger.

**Comment by kilian102201** [comment]
> @kilian102201 some other solutions to your request are to change the default font size in windows or change the scaling. You can find these under Settings > ease of access > make text bigger and settings > ease of access > make everything bigger, respectively.

I don't want to change the overal scaling for my complete windows setup. 
I've installed the files application via the windows store so I can't acces the installation folder and force the application to start with a different type of scaling for example. 

So resizing the font inside the application is still not possible. That's too bad. 

The same goes for dragging / sorting quick access pinned folders is also not possible. 
It's really cool to be able to tuck in the sidebar on the left side but whenever you open a new windows it opens up again. 

So there are still a couple of things that feel off to use it as a daily "driver"

I've tried out the unsupported registry edit to use files as my standard explorer application.
It does work most of the time but it doesn't work when i'm directly downloading something in my browser.
